solidity/deploy/02_deploy_vending_machine.ts

```diff
@@ -17,6 +17,13 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     log: true,
   })
 
+  if (hre.network.tags.tenderly) {
```

Comment:
Let's move tenderly verification to the bottom of the script, below `transferOwnership`.

Reply:
Done, see 0927681ee1e3de4c910fc2e4d6d59a892ccf262b.